src/util/http/MediaTypes.cpp

```diff
@@ -140,7 +140,7 @@ std::vector<MediaTypeWithQuality> parseAcceptHeader(
     antlr4::ANTLRInputStream stream{input};
     AcceptHeaderLexer lexer{&stream};
     antlr4::CommonTokenStream tokens{&lexer};
-    ThrowingErrorListener errorListener_{};
+    ThrowingErrorListener<InvalidSparqlQueryException> errorListener_{};
```

Comment:
This is not a SPARQL parser, but a MediaType parser, so it should use a different exception.

Reply:
Like [this](https://github.com/ad-freiburg/qlever/pull/1044/commits/18f7a01d78eabe3ede863ee00d53ffe3c6f724fb)?